services/agents/mongodb.go

```diff
@@ -160,6 +211,39 @@ func v225Args(exporter *models.Agent, disabledCollectors []string, tdp *models.D
 	return args
 }
 
+func defaultCollectors(collectAll bool, collstatsLimit int32) map[string]collectorArgs {
+	return map[string]collectorArgs{
+		"diagnosticdata": {
+			enabled:     true,
+			enableParam: "--collector.diagnosticdata",
+		},
+		"replicasetstatus": {
+			enabled:     true,
+			enableParam: "--collector.replicasetstatus",
+		},
+		// disabled until we have better information on the resources usage impact
+		"collstats": {
+			enabled:     false || (collectAll && collstatsLimit != 0),
```

Comment:
false || is uneccesary here and in cases below.

Reply:
Fixed. Could you re -review?